Packs/HealthCheck/Dashboards/dashboard-c7f84a7b-341b-4e9d-8911-1df1369d1d69-XSOAR_Status_Review_Dashboard_v6.json

```diff
@@ -0,0 +1,2626 @@
+                "sequenceNumber": 486229062,
+                "shouldCommit": false,
+                "sizeInBytes": 683,
+                "toServerVersion": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "version": 1,
+                "widgetType": "table"
+            },
+            "x": 5,
+            "y": 26
+        },
+        {
+            "forceRange": true,
+            "h": 6,
+            "i": "0e7959d0-101b-11ef-8e4a-9990e7e56f67",
+            "id": "0e7959d0-101b-11ef-8e4a-9990e7e56f67",
+            "reflectDimensions": true,
+            "w": 3,
+            "widget": {
+                "Cache": null,
+                "cacheVersn": 0,
+                "category": "",
+                "commitMessage": "",
+                "created": "2024-05-12T04:40:33.839899004Z",
+                "dataType": "scripts",
+                "dateRange": {
+                    "fromDate": "0001-01-01T00:00:00Z",
+                    "fromDateLicense": "0001-01-01T00:00:00Z",
+                    "period": {
+                        "by": "",
+                        "byFrom": "",
+                        "byTo": "",
+                        "field": "",
+                        "fromValue": null,
+                        "toValue": null
+                    },
+                    "toDate": "0001-01-01T00:00:00Z"
+                },
+                "definitionId": "",
+                "fromServerVersion": "",
+                "id": "69fd82b9-8c42-4fee-8461-d19a2364abc1",
+                "indexName": "testprefix-dmst-common-configuration",
+                "isPredefined": false,
+                "itemVersion": "",
+                "modified": "2024-05-12T04:40:33.845487533Z",
+                "name": "Detached OOTB Automations",
+                "packID": "",
+                "packName": "",
+                "params": {
+                    "tableColumns": []
+                },
+                "prevName": "Detached OOTB Automations",
+                "primaryTerm": 15,
+                "propagationLabels": [
+                    "all"
+                ],
+                "query": "getDetachedAutomations",
+                "sequenceNumber": 486229230,
+                "shouldCommit": false,
+                "sizeInBytes": 635,
+                "toServerVersion": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "version": 1,
+                "widgetType": "table"
+            },
+            "x": 9,
+            "y": 22
+        }
+    ],
+    "name": "XSOAR Status Review Dashboard v6",
+    "period": {
+        "by": "",
+        "byFrom": "days",
+        "byTo": "",
+        "field": "",
+        "fromValue": 30,
+        "toValue": null
+    },
+    "toDate": "0001-01-01T00:00:00Z",
+    "version": -1,
+    "fromVersion": "6.10.0",
+    "toVersion": "6.13.0",
+    "description": "",
```

Comment:
Pls add a description to this dashboard on it's intent - note that you need to add to both of the dashboards.